express/manifest.py

```diff
@@ -163,16 +171,25 @@ def subsets(self) -> t.Mapping[str, Subset]:
             }
         )
 
+    def get_subset(self, name: str) -> Subset:
+        if name not in self._specification["subsets"]:
+            raise ValueError(f"Subset {name} not found in specification")
+
+        return Subset(self._specification["subsets"][name], base_path=self.base_path)
+
     def add_subset(self, name: str, fields: t.List[t.Tuple[str, Type]]) -> None:
         if name in self._specification["subsets"]:
             raise ValueError("A subset with name {name} already exists")
 
         self._specification["subsets"][name] = {
-            "location": f"/{name}/{self.run_id}/{self.component_id}",
-            "fields": {name: {"type": type_.value} for name, type_ in fields},
+            "location": f"/custom_artifact/{self.run_id}/{self.component_id}/{name}",
+            "fields": {name: {"type": type_} for name, type_ in fields},
```

Comment:
why did we remove the `values` from fields

Reply:
Have addressed this by rolling back